### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by daoso** [GrandaddyShmax/audiocraft_plus#12] [Ideas]
Thank you for Audiocraft Plus! Such an amazing webui, thank you for maintaining it and adding new features.

I wonder, do you have a discord server? One solely dedicated to music generation would be extremely useful

**Comment by GrandaddyShmax** [comment]
Thank you ☺
Yes there is server dedicated to musicgen, not mine but I am active there: https://discord.gg/sexDHqDr

**Reply by daoso** [reply]
Thank you, this is just perfect!
and the discord url is quite alluring :D

Love what you are doing, I can feel the passion you have 
I mean, the creative one, not related to the url

Your webui is miles ahead of every other musicgen one,
not to disregard efforts of other developers, I appreciate everyone working
on this still very niche area of music generation
I feel there is still so much untapped potential
lies inside the Meta's model... and you're uncovering it.. so

Thank you <3

**Reply by GrandaddyShmax** [reply]
only now noticed the url 😂
thank you so much, i truly appreciate it 🤗
to think that it just happened spontaneously, never worked that much with python before
just wanted to try it and seems like it ended up being beneficial to others

**Reply by daoso** [reply]
it's more than beneficial :pray: :heart:
